src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -233,7 +233,7 @@ object TezosTypes {
   final case object DoubleEndorsementEvidence extends Operation
   final case object DoubleBakingEvidence extends Operation
   final case object Proposals extends Operation
-  final case object Ballot extends Operation
+  final case class Ballot(ballot: Voting.Vote) extends Operation
```

Comment:
How come this is a case class and not case object like the others?

Reply:
because it has a parameter, like the earlier operations.